node/src/components/consensus.rs

```diff
@@ -56,13 +56,17 @@ pub enum ConsensusMessage {
     /// A request for evidence against the specified validator, from any era that is still bonded
     /// in `era_id`.
     EvidenceRequest { era_id: EraId, pub_key: PublicKey },
+    /// A request for the latest state in the given era.
+    LatestStateRequest { era_id: EraId },
```

Comment:
We could send our own latest honest panorama for that era, in case we have everything.

Reply:
It will take a bit of redoing (the request will have to become a protocol message), but it might actually make more sense.

Reply:
Depends on how high the priority is for that ticket. We can merge it as-is and then schedule a follow-up. You could even start working on it right after this PR is merged. This change is, after all, an important improvement.

Reply:
It's not much work to make it like you're suggesting - I'll just need to rewrite a part of the PR. I think it will actually make more sense afterwards, so I'll do that.